### ISSUE TITLE
batchwise computation

### ISSUE THREAD

**Issue by zhangmozhe** [jeanfeydy/geomloss#28]
Hi, I met error "The size of tensor a (2048) must match the size of tensor b (4096) at non-singleton dimension 1" when I run the following code:

points1 = torch.rand(2, 2048, 3, requires_grad=True).cuda()
points2 = torch.rand(2, 2048, 3).cuda()
geomloss.SamplesLoss(loss='sinkhorn', p=2, blur=0.05)(points1 , points2)

It seems that the code does not process one batch input well. I wonder how to solve it. Thanks!

**Comment by zhangmozhe** [comment]
installation from the current source code solves my problem. Sorry to bother.
